app/views/papers/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <h1>Papers</h1>
 
-<%= link_to 'New Paper', new_paper_path %>
+<%= link_to 'New Paper', new_paper_path %> </br>
```

Comment:
This irritates me a little bit, but I did something equally annoying with the inline style elsewhere.